migration/sql-files/019-authorization-part-2.sql

```diff
@@ -0,0 +1,36 @@
+ALTER TABLE resource DROP CONSTRAINT resource_parent_resource_id_fkey;
+ALTER TABLE identity_role DROP CONSTRAINT identity_role_resource_id_fkey;
+
+ALTER TABLE resource ALTER COLUMN resource_id TYPE varchar;
+ALTER TABLE resource ALTER COLUMN parent_resource_id TYPE varchar;
```

Comment:
Any reason why it is varchar and not uuid ?

Reply:
Yes, the resource_id value can actually be set by the resource server when the resource is registered.  In the case of WIT, they will use the natural key value of the resource that is being created.  It will most likely be a uuid, but it doesn't have to be.  In any case the column is indexed and has a unique constraint, so there are no issues with performance or duplicate key values.

Reply:
Thanks - makes sense!

Reply:
I'm not sure by maybe we should limit the size of varchars? To make sure resource services are not setting some crazy long IDs.

Reply:
+1, what do you think is a reasonable size?